Sources/AppcuesKit/Presentation/Debugger/UIDebugger.swift

```diff
@@ -138,14 +138,14 @@ extension UIDebugger: DebugViewDelegate {
             hide()
         case .open:
             viewModel.ping()
-        case .screenCapture:
-            captureScreen()
+        case let .screenCapture(authorization):
+            captureScreen(authorization: authorization)
         case .show, .close, .reposition:
             break
         }
     }
 
-    private func captureScreen() {
+    private func captureScreen(authorization: Authorization) {
```

Comment:
this function is where things will hook in later and use the auth to make the customer API calls